drivers/npmx/npmx_shell.c

```diff
@@ -52,7 +52,10 @@ static int cmd_charger_termination_voltage_normal_get(const struct shell *shell,
 		npmx_charger_termination_normal_voltage_get(charger_instance, &voltage);
 
 	if (check_error_code(shell, err_code)) {
-		shell_print(shell, "Value: %d mV.", npmx_charger_voltage_convert_to_mv(voltage));
+		uint32_t voltage_mv;
+
+		NPMX_ASSERT(npmx_charger_voltage_convert_to_mv(voltage, &voltage_mv) == true);
```

Comment:
I do not know whether passing a function to the assertion is a good solution. What if the assertion will be empty? The function will never be even compiled.

Reply:
Same as in LDO.  Also, I think shutting all application down due to an error in the convert function is too much.

Reply:
With @wilu-nordic I agree, with @magp-nordic not as much - Imo we should validate it somehow, moreover this assertion can fail only if we will mess up in our functions. User cannot "trigger" this assert, since this conditional statement checks if the voltage value is valid - in other words:
If this assert fail, it's only because of us, and we should fix it as soon as possible.
@masz-nordic any thoughts?

Reply:
I think instead of asserting, shell error should be printed in case of an invalid value.